src/jsdocs/providers.yaml

```diff
@@ -0,0 +1,32 @@
+paths:
+  # Endpoint to authenticate a user
+  /google:
+    # GET operation
+    get:
+      tags:
+        - providers
+
+      summary: Authenticates with Google
+
+      security:
+        - Bearer: []
+
+      responses:
+        "200":

```

Comment:
This file should also probably be named something like `googleSignIn.yaml` so it is clear what it does